docs/en/observability/slo-privileges.asciidoc

```diff
@@ -7,10 +7,21 @@
 
 IMPORTANT: To create and manage SLOs, you need an {subscriptions}[appropriate license].
 
-You can create the following roles for your SLOs:
+You can enable access to SLOs in two different ways:
 
-* <<slo-all-access,*SLO All*>> — Create, edit, and manage SLOs and their historical summaries.
-* <<slo-read-access,*SLO Read*>> — See SLOs and their historical summaries.
+* Creating the following roles, depending on the type of access needed:
+** <<slo-all-access,*SLO Editor*>> — Create, edit, and manage SLOs and their historical summaries.
+** <<slo-read-access,*SLO Viewer*>> — See SLOs and their historical summaries.
+
+* Using the `editor` {ref}/built-in-roles.html[built-in role]. This role grants full access to all features in {kib} (including the {observability} solution)
+and read-only access to data indices. Users assigned to this role can create, edit, and manage SLOs.
++
+[NOTE]
+====
+The `editor` {ref}/built-in-roles.html[built-in role] will grant write access
+to _all_ {kib} apps. If you want to limit access to the SLOs only, you will have
+to manually create and assign the mentioned roles.
+====
```

Comment:
```suggestion
====
The `editor` {ref}/built-in-roles.html[built-in role] grants write access
to _all_ {kib} apps. If you want to limit access to the SLOs only, you have
to manually create and assign the mentioned roles.
====
```
Limiting the use of future tense and preferring the present tense where possible makes the reading lighter.